Code/GraphMol/MolDraw2D/Wrap/rdMolDraw2D.cpp

```diff
@@ -655,7 +659,11 @@ BOOST_PYTHON_MODULE(rdMolDraw2D) {
                RDKit::MolDraw2D::drawOptions,
            python::return_internal_reference<
                1, python::with_custodian_and_ward_postcall<0, 1>>(),
-           "Returns a modifiable version of the current drawing options");
+           "Returns a modifiable version of the current drawing options")
+      .def("SetDrawOptions", RDKit::setDrawOptions,
```

Comment:
Shouldn't this be in the constructor as well?

Reply:
The constructor for what?

Reply:
The constructor for what?

Reply:
I meant something like:

```
do = rdMolDraw2D.MolDrawOptions()
do.useBWAtomPalette()
d = rdMolDraw2D.MolDraw2DSVG(250, 200, options=do)
```

Reply:
Given that you can just assign directly to the result of calling drawOptions() it doesn’t seem like that adds much value?